app/views/sessions/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
```

Comment:
Is this file necessary?

Reply:
Again was created by rails when we ran the generate method for the sessions controller, it could be useful when we add CSS styles